bubble_sort.rb

```diff
@@ -0,0 +1,19 @@
+def bubble_sort(arr)
+  for i in 0..arr.length
+    for j in i + 1..arr.length - 1
+      if arr[i] > arr[j]
+        arr[i], arr[j] = arr[j], arr[i]
+      end
+    end
+  end
+  puts arr
+end
+
+bubble_sort([15, 20, 3, 55, 4, 85, 9, 2, 11, 6, 14])
+
+def bubble_sort_by(arr)
+  arr.sort! { |first_i, second_i| first_i.upcase <=> second_i.upcase }
```

Comment:
- [ ] you were to build a bubble_sort_by method from scratch. not to use the original sort method inside of it.